src/Tasks/Microsoft.CSharp.CurrentVersion.targets

```diff
@@ -100,6 +100,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         <CreateCSharpManifestResourceName
               ResourceFiles="@(EmbeddedResource)"
               RootNamespace="$(RootNamespace)"
+              EnableCustomCulture="$(EnableCustomCulture)"
```

Comment:
Does this not also need the opt-out list?

Reply:
without passing `EnableCustomCulture` explicitly it won't be possible to control custom culture feature - it was also modified in scope of https://github.com/dotnet/msbuild/issues/11446